lib/facilities/multi_client.rb

```diff
@@ -15,7 +16,7 @@ def run(requests)
       requests.map do |req|
         if req.response.timed_out?
           Rails.logger.error "GIS request timed out: #{req.url}"
-          raise Common::Client::Errors::RequestTimeout
+          raise Common::Exceptions::ClientError.new('Facility request timed out', {})
```

Comment:
In which case you might ask why merge this at all? Because it's throwing up annoying errors in sentry due to the wrong exception class.